server/cmwell-cons/src/main/scala/GridSubDiv.scala

```diff
@@ -227,41 +164,38 @@ case class GridSubDiv(user: String,
     val bgAllocations = aloc.bg //DefaultAlocations(1000,1000,512,0)
     val wsAllocations = aloc.ws
     val ctrlAllocations = aloc.ctrl
-
-    val topologyMap = topology.getTopologyMap(ipMappings)
     val homeDir = s"${instDirs.globalLocation}/cm-well"
-    hosts.flatMap { host =>
-      val aliases = topologyMap(host)
+    val casDataDirs = for (i <- 1 to dataDirs.casDataDirs.size)
+      yield {ResourceBuilder.getIndexedName("cas", i)}
 
-      val casSubDivs = for (i <- 1 to dataDirs.casDataDirs.size)
-        yield {
-          CassandraConf(
-            home = homeDir,
-            seeds = getSeedNodes.mkString(","),
-            clusterName = clusterName,
-            resourceManager = casAllocations,
-            snitchType = "GossipingPropertyFileSnitch",
-            ccl_dir = ResourceBuilder.getIndexedName("ccl", i),
-            dir = ResourceBuilder.getIndexedName("cas", i),
-            rowCacheSize = casRowCacheSize,
-            replicationFactor = 3,
-            template = "cassandra.yaml",
-            listenAddress = aliases(i - 1),
-            rpcAddress = aliases(i - 1),
-            sName = s"${ResourceBuilder.getIndexedName("start", i)}.sh",
-            index = i,
-            rs = IpRackSelector(),
-            g1 = g1,
-            hostIp = host
-          )
-
-        }
+    hosts.flatMap { host =>
+      val cas = CassandraConf(
+        home = homeDir,
+        seeds = getSeedNodes.mkString(","),
+        clusterName = clusterName,
+        resourceManager = casAllocations,
+        snitchType = "GossipingPropertyFileSnitch",
+        ccl_dir = "ccl",
+        dir = "cas",
+        rowCacheSize = casRowCacheSize,
+        replicationFactor = 3,
+        template = "cassandra.yaml",
+        listenAddress = host,
+        rpcAddress = host,
+        sName = "start.sh",
+        index = 1,
+        rs = IpRackSelector(),
+        g1 = g1,
+        hostIp = host,
+        casDataDirs = casDataDirs,
+        casDurableWrites = false
```

Comment:
please add same comment as in Grid.scala

Reply:
done